src/vues/fragments/elementsAjoutables/elementsAjoutables.pug

```diff
@@ -0,0 +1,23 @@
+mixin zoneSaisieElementAjoutable(proprietesElement, nom, index)
+  each valeur, cle in proprietesElement
+    input(
+      id = cle + '-' + nom + '-' + index,
+      name = cle + '-' + nom + '-' + index,
+      type = 'text',
+      value != valeur.valeur,
+      placeholder = valeur.valeurExemple
+    )
+
+mixin elementsAjoutables({ identifiantConteneur, nom, proprietesElements = [], texteLienAjouter = 'Ajouter' })
+  div(id = identifiantConteneur)
+    each value, index in proprietesElements
+      label.item-ajoute
+        .icone-suppression
+        div(class = 'element-' + nom + '-' + index)
+          +zoneSaisieElementAjoutable(value, nom, index)
```

Comment:
Quelque chose ne colle pas ici… on appelle le mixin avec une valeur atomique `value`, alors que le mixin se comporte comme si c'était une liste d'éléments (cf. ligne 2 `each… in…`). J'ai loupé quelque chose ?

Reply:
c'est dans le cas où l'on a plusieurs champs inputs 

```javascript
[
  {
    nom: {
      valeur: "Jean",
      valeurExemple: "Nom"
    },
    fonction: {
      valeur: "Directeur",
      valeurExemple: "Fonction"
    }
  },
  {
    nom: {
      valeur: "Jacques",
      valeurExemple: "Nom"
    },
    fonction: {
      valeur: "Vice-directeur",
      valeurExemple: "Fonction"
    }
  }
]
```

Reply:
Ok – j'ai compris. Je me suis fait avoir par le nom `proprietesElements` – j'ai cru que ça représentait `['nom', 'fonction']`… alors que pas du tout 😅 Et le nouveau `propriete` (à la place du `value` initial) n'aide pas à mieux comprendre 😓

En, fait, j'ai l'impression qu'on a les `donneesElements` (toutes les données pour tous les éléments), puis pour chaque `donneesElement` (toutes les données pour un élément) on appel `zoneSaisieElementAjoutable(donneesElement, nom, index)`. Qu'en penses-tu ?

Reply:
j'ai mis des donneesElements et des donneesElement 👍🏼